product_pricelist_direct_print/wizards/product_pricelist_print.py

```diff
@@ -268,10 +272,15 @@ def force_pricelist_send(self):
 
     @api.model
     def _get_sale_order_domain(self, partner):
-        return [
+        domain = [
             ("state", "not in", ["draft", "sent", "cancel"]),
             ("partner_id", "child_of", partner.id),
         ]
+        if self.product_selling_date_threshold:
+            domain = expression.AND(
+                [domain, [("date_order", ">=", self.product_selling_date_threshold)]]
+            )
```

Comment:
```suggestion
            domain.append(("date_order", ">=", self.product_selling_date_threshold))
```
I know I review the PR in the previous version and I didn't put this suggestion, but seeing that it's a very simple domain we can save ourselves the use of expression.AND.

Reply:
expression.AND is more explicit. And that is exactly the objective of this function.